Dockerfile.plugins.demo2

```diff
@@ -0,0 +1,22 @@
+# This image is used for testing OpenShift Console dynamic plugin capabilities.
```

Comment:
Why do we need two different Dockerfiles for the demo plugin?

Reply:
This is created by Christian. Hey @christianvogt , do you think we still need this Dockerfile?

Reply:
@spadgett i created this separate dockerfile because it greatly simplified testing. The normal docker file takes forever to build. Running a local build for the demo plugin is quick. This dockerfile assumes a pre-built plugin and simply packages it.

Reply:
It's a little weird to have two Dockefiles, but I guess it's OK. Long term we should try to figure out why the build is so slow and try to fix that so it's not needed.